client/src/components/Pagination.jsx

```diff
@@ -85,6 +85,7 @@ export default function Pagination(props) {
                 // On screens narrower than 768, show the settings popup as a
                 // modal rather than a dropdown menu
                 modal={typeof window !== "undefined" && window.innerWidth < 768}
+                position="bottom left"
```

Comment:
Ah good catch, I moved it because I only tested it against the left side, where it runs into the same problem when center aligned. Should it switch over to modal view earlier then? Ideally it would change directions based on how close it is to the edge of the screen.

Reply:
Fixed this by setting `keepTooltipInside="body"`, [docs here](https://react-popup.elazizi.com/react-tooltip#smart-tooltip-position). I also set `repositionOnResize` which could affect performance since I assume it's adding a listener to the window but it's probably worth it to make sure the tooltip never gets cut off.

The modal view may not be necessary at all with this change, since the popup easily fits on even small screens:
![image](https://user-images.githubusercontent.com/7061296/162642935-96a6abf5-7c3a-4f0b-95e7-3cd5b2861f57.png)

Reply:
> Ideally it would change directions based on how close it is to the edge of the screen.

True, I'm a little bummed the library doesn't handle that automatically. Perhaps it's something I can enable if I read the documentation a bit more carefully. I think 768 is a good time to switch to a modal view, and I don't really want to do it earlier just to work around a bug.

I'll create a new issue for it and see if I can solve it properly the next time I have a proper sit down with Clipface.

I think I'd prefer to have it centered meanwhile, if for no other reason than I think it looks nice :smile:

Reply:
Wait, what am I saying. I just had a look at your last commit and you've already solved it properly, it seems like :laughing:

Reply:
Yeah I found the solution after I posted the first comment hehe

Reply:
> The modal view may not be necessary at all with this change, since the popup easily fits on even small screens

True, but in my limited UX experience, modal popups is more common and expected on mobile devices. For example, if you click a select box in a browser on a cellphone, you'll almost certainly get a modal popup, not a dropdown menu.

Could definitely decrease the screen width threshold down a bit though, maybe `576px`.